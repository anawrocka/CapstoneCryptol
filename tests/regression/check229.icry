:cd
:cd non-existent_directory