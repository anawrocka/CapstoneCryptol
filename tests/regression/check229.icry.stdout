Loading module Cryptol
[error] :cd requires a path argument

Directory `non-existent_directory' not found or not a directory
